ng2-components/ng2-activiti-analytics/src/components/widgets/duration/duration.widget.html

```diff
@@ -5,6 +5,7 @@
                    type="text"
                    pattern="-?[0-9]*(\.[0-9]+)?"
                    [attr.id]="field.id"
+                   [value]="field.value"
```

Comment:
the problem is that you are unit testing it hoping that value will always be equal to ngModel, but if  someone changes the value passed in ngModel your unit test will keep been green because you are testing the other filed....